stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -678,34 +678,66 @@ public final class _DataStorage {
     }
     
     @inline(__always)
-    public func mutableCopy() -> _DataStorage {
+    public func mutableCopy(_ range: Range<Int>) -> _DataStorage {
```

Comment:
the inline case that is interesting here is the swift case, which is quite small. I wish there was a way to only inline part of the switch here.